egs2/rats/spk1/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -o pipefail
+
+
+spk_config=conf/train_RawNet3.yaml
+
+train_set="rats_train"
+valid_set="rats_test"
+cohort_set="rats_test"
```

Comment:
can you remove this line? I believe your experiment didn't use score normalization.
It is not ideal to note the test set as cohort set as you'd know.

Reply:
sorry, only L11